notebooks/community/model_garden/model_garden_pytorch_mistral_peft_tuning.ipynb

```diff
@@ -0,0 +1,1460 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    # model = AutoModelForCausalLM.from_pretrained(model_id, quantization_config=bnb_config, device_map="auto")
</pre><p>Do we need to keep the dead code? If so, it's better to explain </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2645/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>